replication-groomer/src/main/java/com/octo/replication/groomer/ReplicationMetacardGroomerPlugin.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.octo.replication.groomer;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import ddf.catalog.Constants;
+import ddf.catalog.content.data.ContentItem;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.operation.CreateRequest;
+import ddf.catalog.operation.Request;
+import ddf.catalog.operation.UpdateRequest;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.plugin.groomer.AbstractMetacardGroomerPlugin;
+import ddf.security.SecurityConstants;
+import ddf.security.SubjectOperations;
+import java.io.Serializable;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Map;
+import java.util.Map.Entry;
+import org.apache.shiro.subject.Subject;
+import org.codice.ddf.configuration.SystemInfo;
+import org.codice.ddf.platform.util.uuidgenerator.UuidGenerator;
+import org.codice.ditto.replication.api.Replication;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Copied of the StandardMetacardGroomer and updated for replication purposes. This groomer replaces
+ * the StandardMetacardGroomer and should be the only groomer running in the system.
```

Comment:
This is a good comment. Not sure if it makes sense to have documented somewhere else eventually so people know to not have the StandardMetacardGroomer running.

Reply:
Agree. Not sure where we are documenting things like that